packages/rpcs/wallet_handleUnfinishedCFXTx/index.js

```diff
@@ -218,9 +217,9 @@ export const main = ({
               })
 
             defs({
-              failed: failed && {errorType, err},
-              isDuplicateTx,
-              keepTrack: !failed,
+              failed: shouldDiscard && {errorType, err},
```

Comment:
if the error case is `txPoolFull`, the shouldDiscard will also be `false`, in this case, the `failed` value will not get the `{errorType, err}`. I think `txPoolFull` should get the error?

Reply:
https://github.com/Conflux-Chain/helios/blob/c9d4165bf2100231a5d040bf0c0b8ae4997c2b48/packages/conflux-tx-error/index.js#L8